app/uk/gov/hmrc/merchandiseinbaggage/model/core/DeclarationJourney.scala

```diff
@@ -153,6 +153,7 @@ case class DeclarationJourney(
 
 object DeclarationJourney {
   implicit val format: OFormat[DeclarationJourney] = Json.format[DeclarationJourney]
+//  implicit val dateFormat: Format[LocalDateTime] = MongoJavatimeFormats.localDateTimeFormat
```

Comment:
remove this commented like and use this one https://github.com/hmrc/trusts-store/blob/main/app/models/MongoDateTimeFormats.scala#L26-L54